launch_testing/launch_testing/pytest/hooks.py

```diff
@@ -110,8 +110,16 @@ def pytest_pycollect_makemodule(path, parent):
         if module is not None:
             return module
     if path.basename == '__init__.py':
-        return pytest.Package(path, parent)
-    return pytest.Module(path, parent)
+        try:
+            # since pytest 5
+            return pytest.Package.from_parent(parent, fspath=path)
+        except AttributeError:
```

Comment:
I would add a todo, saying that this can be dropped after all our platforms are using `5.4.0` or higher.